santasflor.test.js

```diff
@@ -0,0 +1,6 @@
+const santasFloor = require('./index.js');
+
+test('sant shoud be in 3º flor', () => {
+  const inputValue = '((('
+  expect(santasFloor(inputValue)).toBe('3');
```

Comment:
You are comparing string with integer. 
Try this:
`expect(santasFloor(inputValue)).toBe(3);`